src/main/java/core/basesyntax/service/FruitServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+
+public class FruitServiceImpl implements FruitService {
```

Comment:
This service works with report
```suggestion
public class ReportServiceImpl implements ReportService {
```